changeset:   6214:ccfaf1c37a56
user:        Panu Matilainen <pmatilai@redhat.com>
date:        Mon Aug 06 14:47:43 2007 +0300
files:       installplatform rpmrc.in
description:
Couple of ARM-related typo fixes from Lennert Buytenhek.


diff -r f0a6328331a5 -r ccfaf1c37a56 installplatform
--- a/installplatform	Mon Aug 06 14:47:10 2007 +0300
+++ b/installplatform	Mon Aug 06 14:47:43 2007 +0300
@@ -32,7 +32,7 @@ case "$arch" in
 case "$arch" in
   i[3456]86|pentium[34]|athlon) SUBSTS='s_i386_i386_ s_i386_i486_ s_i386_i586_ s_i386_i686_ s_i386_pentium3_ s_i386_pentium4_ s_i386_athlon_' ;;
   alpha*) SUBSTS='s_alpha_alpha_ s_alpha_alphaev5_ s_alpha_alphaev56_ s_alpha_alphapca56_ s_alpha_alphaev6_ s_alpha_alphaev67_' ;;
-  arm*) SUBSTS='s_arm_armv3l_ s_arm_arm4l_ s_arm_armv4tl_ s_arm_armv5tel_ s_arm_armv5tejl_ s_arm_armv6l_' ;;
+  arm*) SUBSTS='s_arm_armv3l_ s_arm_armv4l_ s_arm_armv4tl_ s_arm_armv5tel_ s_arm_armv5tejl_ s_arm_armv6l_' ;;
   sparc*) SUBSTS='s_sparc\(64\|v9\)_sparc_ s_sparc64_sparcv9_;s_sparc\([^v]\|$\)_sparcv9\1_ s_sparcv9_sparc64_;s_sparc\([^6]\|$\)_sparc64\1_' ;;
   powerpc*|ppc*) SUBSTS='s_ppc64_ppc_ s_ppc\([^6ip]\|$\)_ppc64\1_ s_ppc\([^6ip]\|$\)_ppciseries_ s_ppc\([^6ip]\|$\)_ppcpseries_ s_ppc\([^6ip]\|$\)_ppc64iseries_ s_ppc\([^6ip]\|$\)_ppc64pseries_' ;;
   s390*) SUBSTS='s_s390x_s390_ s_s390\([^x]\|$\)_s390x\1_' ;;
diff -r f0a6328331a5 -r ccfaf1c37a56 rpmrc.in
--- a/rpmrc.in	Mon Aug 06 14:47:10 2007 +0300
+++ b/rpmrc.in	Mon Aug 06 14:47:43 2007 +0300
@@ -230,7 +230,7 @@ buildarchtranslate: armv4l: armv4l
 buildarchtranslate: armv4l: armv4l
 buildarchtranslate: armv4tl: armv4tl
 buildarchtranslate: armv5tel: armv5tel
-buildarchtranslate: armv5tejl: armv4tejl
+buildarchtranslate: armv5tejl: armv5tejl
 buildarchtranslate: armv6l: armv6l
 
 buildarchtranslate: atarist: m68kmint

